WordPress/Classes/ViewRelated/NUX/NUXCollectionViewController.swift

```diff
@@ -14,6 +14,10 @@ class NUXCollectionViewController: UICollectionViewController, NUXViewController
         }
     }
 
+    override var supportedInterfaceOrientations: UIInterfaceOrientationMask {
```

Comment:
Looks good.

I see we're doing this in a few places. May be a good candidate to move into the `NUX*ViewController` base classes at some point.